src/app/Shared/Services/CommandChannel.service.tsx

```diff
@@ -27,21 +27,27 @@ export class CommandChannel {
         (url: any) => this.clientUrlSubject.next(url.clientUrl),
         (err: any) => this.logError('Client URL configuration', err)
       );
+    
+    const getDatasourceURL = fromFetch(`${this.apiSvc.authority}/api/v1/grafana_datasource_url`)
+    .pipe(concatMap(resp => from(resp.json())));
+    const getDashboardURL = fromFetch(`${this.apiSvc.authority}/api/v1/grafana_dashboard_url`)
+    .pipe(concatMap(resp => from(resp.json())));
 
-    fromFetch(`${this.apiSvc.authority}/api/v1/grafana_datasource_url`)
-      .pipe(concatMap(resp => from(resp.json())))
-      .subscribe(
-        (url: any) => this.grafanaDatasourceUrlSubject.next(url.grafanaDatasourceUrl),
-        (err: any) => this.logError('Grafana Datasource configuration', err)
-      );
-
-    fromFetch(`${this.apiSvc.authority}/api/v1/grafana_dashboard_url`)
-      .pipe(concatMap(resp => from(resp.json())))
+    fromFetch(`${this.apiSvc.authority}/health`)
+      .pipe(
+        concatMap(resp => from(resp.json())))
```

Comment:
I think you could just `map(resp => resp.json())` here - no need to make the json into an Observable and then immediately concat it in this case, from what I can tell.